src/data/animations/anim-config.ts

```diff
@@ -86,23 +86,70 @@ export class AnimConfig {
     return Array.from(backgrounds.values());
   }
 }
+
+/**
+ * Contains data for a single frame of an asset in a battle animation
+ * @see {@linkcode BattleAnim}
+ */
 export class AnimFrame {
+  /**
+   * The x-position of the sprite, relative
+   * to the frame's {@linkcode AnimFocus | focal point}
+   */
   public x: number;
+  /**
+   * The y-position of the sprite, relative
+   * to the frame's {@linkcode AnimFocus | focal point}
+   */
   public y: number;
+  /** Horizontal scale factor (%) */
   public zoomX: number;
+  /** Vertical scale factor (%) */
   public zoomY: number;
+  /** Rotation angle (degrees) */
   public angle: number;
+  /** If `true`, inverts the sprite horizontally */
   public mirror: boolean;
+  /** Whether or not the sprite is visible */
   public visible: boolean;
+  /**
+   * The blend mode to specify how the sprite is rendered on the canvas
+   * @see {@link https://docs.phaser.io/api-documentation/constant/blendmodes}
+   */
   public blendType: AnimBlendType;
+  /**
+   * The type of sprite affected by this frame
+   * @see {@linkcode AnimFrameTarget}
+   */
   public target: AnimFrameTarget;
+  /** If {@linkcode target} is "graphic", specifies the sprite index for the frame */
   public graphicFrame: number;
+  /** The alpha value for the animated sprite */
   public opacity: number;
+  /** Unused */
   public color: number[];
+  /** The animated sprite's tone (RGBA) */
   public tone: number[];
+  /** Unused */
   public flash: number[];
+  /**
+   * If the item list for a frame is smaller than previous frames,
+   * graphics are assumed to be deleted in reverse item order.
+   * This, if `true`, prevents that for this item in future frames.
+   */
   public locked: boolean;
+  /**
+   * The depth or z-position of the animated sprite
+   * - 0 is behind all other sprites (except BG)
+   * - 1 is on top of player field
+   * - 3 is on top of both fields
+   * - 5 is on top of player sprite
+   */
   public priority: number;
+  /**
+   * The {@linkcode AnimFocus} specifying the reference point for
```

Comment:
```suggestion
   * The {@linkcode AnimFocus} specifying the point of origin for
```
~~Stolen from~~ Inspired by GoDot naming